packages/projection-typeorm/src/util.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  AlreadyHasActiveConnectionError,
+  CannotConnectAlreadyConnectedError,
+  CannotExecuteNotConnectedError,
+  CannotGetEntityManagerNotConnectedError,
+  ConnectionIsNotSetError,
+  ConnectionNotFoundError,
+  NoConnectionForRepositoryError,
+  NoConnectionOptionError,
+  NoNeedToReleaseEntityManagerError,
+  NoVersionOrUpdateDateColumnError,
+  PersistedEntityNotFoundError,
+  PessimisticLockTransactionRequiredError,
+  QueryRunnerAlreadyReleasedError,
+  QueryRunnerProviderAlreadyReleasedError,
+  TransactionAlreadyStartedError,
+  TransactionNotStartedError
+} from 'typeorm';
+
+// Might have to adjust this list - classes were picked based on names and their doc comments
```

Comment:
Anything need to be done here, or is it a clue for a future investigation :smile: ?

Reply:
Just a clue. We'll need to extend this as we use new features (create new kind of db objects). Default typeorm naming convention isn't very readable, it uses hashes to name some things.